features/font-variant-ligatures.yml

```
@@ -0,0 +1,16 @@
+name: Font variant settings for ligatures
+description: The \`font-variant-ligatures\` CSS property sets how characters can be visually combined for readability or stylistic reasons.
```

Also I'm not sure I mentioned this anywhere: the description field is Markdown and gets converted to HTML when we publish the package.

```suggestion
description: The `font-variant-ligatures` CSS property sets how characters can be visually combined for readability or stylistic reasons.
```